setup.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
+import 'dotenv/config';
+import { User } from './src/moduls/User.js';
```

Comment:
The import path for `User.js` is incorrect. The directory `moduls` should be `modules` as per the file structure.